maths-club-pack/content/facilitator/gabriels-problem.md

```diff
@@ -0,0 +1,92 @@
+# Gabriel's Problem
```

Comment:
These were tricky notes, it's much harder to get images and text arranged in markdown than in word. I think you made the right decision to simply put some of the images after the text (easier to view on a small screen), and I really like the use of the table at the end also to organise. Thanks!